pkgdb2client/cli.py

```diff
@@ -781,11 +778,8 @@ def main():
         LOG.setLevel(logging.INFO)
 
     if arg.test:
-        global fasclient, pkgdbclient
+        global pkgdbclient
         print "Testing environment"
-        fasclient = AccountSystem(
```

Comment:
Also, there doesn't seem to be a way to override the URLs to use test urls in ``pkgdb2client/utils.py``.  If you can change them here with a switch it would be nice to change them there too.